 <launch>
    <!-- 必须先定义agvId -->

    <param name="agvId" value="$(env agvId)"/>

    <node pkg="mqtt_comm" type="iot_comm" name="iot_comm" output="screen"/>

 </launch>